js/src/components/lineage/LineageViewTopBar.tsx

```diff
@@ -336,7 +336,7 @@ const ControlItem = (props: {
 }) => {
   return (
     <Box style={props.style} maxWidth="300px">
-      <Box fontSize="8pt">{props.label || <>&nbsp;</>}</Box>
+      <Box fontSize="8pt">{props.label ?? <>&nbsp;</>}</Box>
```

Comment:
Please keep the original one.
`props.label` can be `""`, and the behavior would change here.

Reply:
done, reverted